node_modules/gh-orgunit/lib/api.js

```diff
@@ -577,6 +580,145 @@ var deleteOrgUnitSeries = module.exports.deleteOrgUnitSeries = function(ctx, id,
+
+    var validator = new Validator();
+    validator.check(id, {'code': 400, 'msg': 'A valid organisational unit id must be provided'}).isInt();
+    if (validator.hasErrors()) {
+        return callback(validator.getFirstError());
+    }
+
+    // Get the organisational unit's calendar
+    _getOrgUnitCalendar(id, null, null, function(err, events, orgUnit) {
+        if (err) {
+            return callback(err);
+        }
+
+        // Return the events as iCal
+        var calendarInfo = _getCalendarInfo(ctx, orgUnit, 'ical');
+        var ical = CalendarUtil.eventsToICal(calendarInfo, events);
+        return callback(null, ical);
+    });
+};
+
+/**
+ * Get the calendar for an organisational unit in RSS format
+ *
+ * @param  {Context}        ctx                 Standard context containing the current user and the current app
+ * @param  {Number}         id                  The id of the organisational unit to get the calendar for in RSS format
+ * @param  {Function}       callback            Standard callback function
+ * @param  {Object}         callback.err        An error object, if any
+ * @param  {String}         callback.rss        The requested organisational unit calendar in RSS format
+ */
+var getOrgUnitCalendarRss = module.exports.getOrgUnitCalendarRss = function(ctx, id, callback) {
+    // Ensure that the organisational unit id is a valid number
+    id = GrasshopperUtil.getNumberParam(id);
+
+    var validator = new Validator();
+    validator.check(id, {'code': 400, 'msg': 'A valid organisational unit id must be provided'}).isInt();
+    if (validator.hasErrors()) {
+        return callback(validator.getFirstError());
+    }
+
+    // Get the organisational unit's calendar
+    _getOrgUnitCalendar(id, null, null, function(err, events, orgUnit) {
+        if (err) {
+            return callback(err);
+        }
+
+        // Return the events as iCal
+        var calendarInfo = _getCalendarInfo(ctx, orgUnit, 'rss');
+        var rss = CalendarUtil.eventsToRSS(calendarInfo, events);
+        return callback(null, rss);
+    });
+};
+
+/**
+ * Get the calendar for an organisational unit
+ *
+ * @param  {Number}         id                  The id of the organisational unit to get the calendar for
+ * @param  {String}         [start]             The timestamp (ISO 8601) from which to get the calendar for the organisation unit
+ * @param  {String}         [end]               The timestamp (ISO 8601) until which to get the calendar for the organisation unit
+ * @param  {Function}       callback            Standard callback function
+ * @param  {Object}         callback.end        An error object, if any
+ * @param  {Event[]}        callback.events     The requested organisational unit calendar
+ * @param  {OrgUnit}        callback.orgUnit    The organisational unit
+ * @api private
+ */
+var _getOrgUnitCalendar = function(id, start, end, callback) {
+    OrgUnitDAO.getOrgUnit(id, false, function(err, orgUnit) {
```

Comment:
Shouldn't this go through the API to check whether the current user has access to the org unit ? (e.g. org unit from a different app)